requirements/frozen.py2.txt

```diff
@@ -308,7 +308,7 @@ Werkzeug==0.14.1
 widgetastic.core==0.21.6
 widgetastic.patternfly==0.0.37
 widgetsnbextension==3.2.1
-wrapanapi==3.0.23
+wrapanapi==3.0.24
```

Comment:
Please make this update across all the frozen files, or remove.

Reply:
Requirements updated by running `python freeze_all.py --upgrade-only wrapanapi`.

Reply:
(discussed offline a bit) The freeze_all brought in new requirements from local env that we don't want.

they're now limited to what packages we want - but we need these updates in both py2 and py3 files.